src/Api/Api.test.tsx

```diff
@@ -0,0 +1,42 @@
+import Api, { Url } from './Api';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+const emptyResponse = {
+  docs: [],
+  total: 0,
+  limit: 10,
+  page: 1,
+  pages: 0,
+};
+
+const server = setupServer(
+  rest.get(Url.allMovies, (req, res, ctx) => {
+    return res(ctx.status(200), ctx.json(emptyResponse));
+  })
+);
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+describe('Api, getMovies', () => {
+  it('should return empty array for unknown movie', async () => {
+    const data = await Api.getMovies('rrrrrrr');
+    expect(data).toEqual(emptyResponse.docs);
+  });
+});
+
+describe('Api, getMovieById', () => {
```

Comment:
u can use one describe for API, because this methods work with one instance

Reply:
corrected